docker-compose.yml

```diff
@@ -1,10 +1,13 @@
-version: "3.8"
-
 services:
   development:
     build: .
     stdin_open: true
     tty: true
+    develop:
+      watch:
+        - action: rebuild
+          path: "./"
+          target: "./forknacular"
```

Comment:
This does not appear to be compatible with [`podman`](https://podman.io), which seemingly only supports a subset of what docker compose has to offer.
Since we recently came to rely on `podman` there needs to be an alternative solution. :/

Reply:
I don't understand what you like to achieve with this. What is not working? I started the project with podman and i can change the index.erb for example and this i directly see.

Reply:
Ah ok, got it, changes in the code are not effected.